src/net/mod.rs

```diff
@@ -724,10 +724,13 @@ pub struct PoxInvData {
     pub pox_bitvec: Vec<u8>, // a bit will be '1' if the node knows for sure the status of its reward cycle's anchor block; 0 if not.
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub struct BlocksDatum(pub ConsensusHash, pub StacksBlock);
```

Comment:
Is there a reason for this particular refactoring?  If it's necessary, can you call it something more specific to indicate that the Stacks block is paired to a particular PoX history (which is the reason we include the consensus hash)?  Maybe `UniqueStacksBlock`?

Reply:
Yes -- the `StacksMessageCodec` trait is defined in `stacks-common`, which means it's a foreign trait, so we cannot implement it for a tuple that contains a foreign type (in this case, this would have been `(ConsensusHash, StacksBlock)`, with the `ConsensusHash` being the foreign type).